README.md

```diff
@@ -12,9 +12,22 @@ Requirements
 
 Some required tools for compiling and testing libstorage-ng are:
 
-gcc-c++, boost-devel, libxml2-devel, libtool, swig >= 3.0.3 and != 3.0.8
-(from [YaST:storage-ng](https://build.opensuse.org/project/show/YaST:storage-ng)),
-doxygen, python-devel, ruby, ruby-devel, rubygem-test-unit
+gcc-c++ boost-devel libxml2-devel libtool doxygen graphviz python-devel ruby ruby-devel
+swig >= 3.0.3 and != 3.0.8 (from [YaST:storage-ng](https://build.opensuse.org/project/show/YaST:storage-ng))
+
+In addition to the previous packages, add these distribution-specific packages as well.
+
+For openSUSE 13.2 and 13.1:
+
+rubygem-test-unit
+
+For Leap, Tumbleweed, SLES:
```

Comment:
For Tumbleweed ruby2.1-rubygem-test-unit is not required.

Leap and SLES are moving targets, e.g. Leap 42.1 or 42.2, SLES11-SP4 or SLES12-SP2?